Code/GraphMol/MolDraw2D/Wrap/rdMolDraw2D.cpp

```diff
@@ -62,7 +62,21 @@ std::map<int, double> *pyDictToDoubleMap(python::object pyo) {
   }
   return res;
 }
+
+void pyListToColourVec(python::object pyo, std::vector<DrawColour> &res) {
```

Comment:
I guess that, just to be careful, you want to do `res.clear()` here first.
You could also just do a `resize()` to the length of the pyo and then use assignment instead of push_back.